numba/tests/support.py

```diff
@@ -52,12 +52,18 @@
 from numba.core.datamodel.models import OpaqueModel
 from numba.pycc.platform import external_compiler_works
 
-
```

Comment:
Should this be removed? There's usually 2 blank lines after `import` decls?

Reply:
flake8 didn't complain when i removed the file from the ignore list.

Reply:
ok, thanks for checking.